js/models/languagesMd.js

```diff
@@ -253,9 +256,9 @@ module.exports = Backbone.Model.extend({
         ReadOnly: "(This Field is Read Only)",
         HandleResolver: "Handle Resolver",
         moderatorSettings: {
-          ProvideResolution: "Provide dispute resolution",
+          DisputeResolution: "Dispute Resolution",
           ServiceFee: "Service fee",
-          ServiceFeeNote: "Percentage of transaction price (max 25, step 0.01, min 0)"
+          ServiceFeeNote: "Min: 0%, Max: 25%"
```

Comment:
I added some JS to prevent the user from adding more than 2 decimals. I tested a bit and it seems to be working well, but you should definitely take a closer look at it to make sure I didn't miss anything.